### PR TITLE
Add Microsoft.Web/connections and Microsoft.Web/connectionGateway resource schemas

### PR DESCRIPTION
**Pull request by swylezol**
…ource schemas

### CODE REVIEW

File: schemas/2016-06-01/Microsoft.Web.json [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2016-06-01"
+          ]
+        },
+        "id": {
+          "type": "string",
+          "description": "The resource id."
+        },
```
Comment (TianoMS) [inline schemas/2016-06-01/Microsoft.Web.json]: please remove Id property.

File: schemas/2016-06-01/Microsoft.Web.json [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
```
Comment (TianoMS) [inline schemas/2016-06-01/Microsoft.Web.json]: same here.

File: schemas/2016-06-01/Microsoft.Web.json (hunk 1 above)
Comment (swylezol) [inline schemas/2016-06-01/Microsoft.Web.json]: fixed (although it was a copy paste from existing resource schemas)

File: schemas/2016-06-01/Microsoft.Web.json (hunk 2 above)
Comment (swylezol) [inline schemas/2016-06-01/Microsoft.Web.json]: fixed

**Comment by TianoMS** [comment]
looks good.

**Pull request merged**